### ISSUE TITLE
Online indicator on messages list

### ISSUE THREAD

**Issue by almejo** [tim-sueberkrueb/ubuntu-hangups#58]
The left sidebar with contacts have little green dots showing if the contact is online. but when in phone mode and inside a conversation I can not tell if the contact is online or not...

I think if when only the messages view is visible you could include the picture of the user and the little dot

**Comment by timsueberkrueb** [comment]
I'm have implemented this a little bit different by setting the status text to "Online" if the user is online and not typing. What I'd also like to add is something like "Last seen ...". This depends on https://github.com/tdryer/hangups/issues/171. 
@tdryer It would be awesome if you could implement this feature :)

**Comment by tdryer** [comment]
> This depends on tdryer/hangups#171

"Last seen" and read timestamps are separate features. I think it's already possible to implement to "last seen" using `Client.query_presence` and `FIELD_MASK_LAST_SEEN`, although this isn't exposed in the `User` or `Conversation` classes yet.

**Comment by timsueberkrueb** [comment]
@tdryer yeah, you're right. Unless it's supposed to be the "Last seen" for this conversation only. Sorry for me always mixing those up ;)
